src/terraform-resource/in/in.go

```diff
@@ -146,6 +147,13 @@ func (r Runner) inWithBackend(req models.InRequest, tmpDir string) (models.InRes
 		return models.InResponse{}, err
 	}
 
+	if req.Params.OutputJSONPlanfile {
```

Comment:
I would move this up higher to just after the Init call. The real environment statefile might not exist yet if you're generating a plan file for a completely fresh environment.